SecurityPkg/Test/Mock/Library/GoogleTest/Ppi/MockTcgPpiLib/MockTcgPpiLib.cpp

```diff
@@ -0,0 +1,15 @@
+/** @file MockTcgPpiLib.cpp

```

Comment:
Please move these mock definitions to the header file and delete the cpp and inf file.

Reply:
Hi Vivian,

I just pushed a new commit as you suggested.
Please help to review it. Thank you.

Regards,
Paddy